source/plugin/messagechannels.rst

```diff
@@ -0,0 +1,118 @@
+================
+Message Channels
+================
+
+In Sponge, every object that messages may be sent to implements the ``MessageReceiver`` interface. A ``MessageChannel``
+is a device that may send messages to an arbitrary number of ``MessageReceiver``\ s and even perform actions like
+recipient-specific formatting of the message.
+
+Selecting Message Recipients
+============================
+
+Within the ``MessageChannel`` interface there are constants and static methods that can be used to obtain or create
```

Comment:
This isn't the only way - you can extend `MessageChannel`, `MutableMessageChannel`, `AbstracctMutableMessageChannel`, etc.